web/client/src/library/pages/docs/Docs.tsx

```diff
@@ -45,59 +39,52 @@ export default function PageDocs(): JSX.Element {
 
   return (
     <Container.Page>
-      <div className="p-4 flex flex-col w-full h-full overflow-hidden">
-        {isArrayNotEmpty(filtered) && (
-          <Search
-            models={filtered}
-            search={search}
-            setSearch={setSearch}
-          />
-        )}
-        {activePlan != null && (
-          <div className="w-full p-4">
-            <TasksOverview tasks={activePlan.tasks}>
-              {({ total, completed, totalBatches, completedBatches }) => (
-                <>
-                  <TasksOverview.Summary
-                    environment={environment.name}
-                    planState={planState}
-                    headline="Most Recent Plan"
-                    completed={completed}
-                    total={total}
-                    totalBatches={totalBatches}
-                    completedBatches={completedBatches}
-                    updateType={
-                      activePlan.type === EnumPlanApplyType.Virtual
-                        ? 'Virtual'
-                        : 'Backfill'
-                    }
-                    updatedAt={activePlan.updated_at}
-                  />
-                </>
-              )}
-            </TasksOverview>
-          </div>
-        )}
-        <SplitPane
-          className="flex w-full h-full overflow-hidden mt-8"
-          sizes={[25, 75]}
-          minSize={0}
-          snapOffset={0}
-        >
-          <div className="py-4 w-full">
-            {models.size > 0 && (
-              <SourceList
-                models={models}
-                filter={filter}
-                setFilter={setFilter}
-              />
-            )}
+      {models.size === 0 ? (
+        <div className="p-4 flex flex-col w-full h-full overflow-hidden">
+          <div className="flex justify-center items-center w-full h-full">
+            <div className="center">
+              <h3 className="text-lg mb-4">
+                Still Waiting For Models To Load...
+              </h3>
+              <p>It should take few more seconds.</p>
+              <p>
+                If it takes too long, probably, there is a problem with
+                connection.
```

Comment:
The UI and API both run locally so the issue wouldn't be with their connection, right?

Reply:
meant with connection between UI and API, need to rephrase